protected/views/dataset/view.php

```diff
@@ -19,20 +19,18 @@
                 <div class="subsection">
                     <div class="media">
                         <div class="media-left">
-                                        <?php if($model->image) {
-                $url = $model->getImageUrl() ? $model->getImageUrl(): $model->image->image('image_upload');
-
-                ?>
-            <a href="<?= $url ?>" >
-                <?= CHtml::image($url, $model->image->tag ,
-                    array(
-                        'class'=>'media-object',
-                        'title'=>$model->image->tag.' License: '.$model->image->license.' Source: '.$model->image->source.' Photographer: '.$model->image->photographer
-                    )
-                ); ?>
-            </a>
-            <?php } ?>
-
+                            <?php if($model->image) {
```

Comment:
These changes in `protected/views/dataset/view.php` are not related to the icons that link out to citation resources? It seems this part of the code is for displaying the dataset image.

Reply:
The content of the code has not been refactored, except the alignment of the code, which would make it more readable. But you are right, maybe better do it in #895 .